bin/nextcloud-domain.sh

```diff
@@ -11,9 +11,12 @@ done
 # wicd service finishes before completing DHCP
 while :; do
   local_ip="$(get_ip)"
-  pub_ip="$(curl -m4 icanhazip.com 2>/dev/null)"
 
-  [[ "$pub_ip"   != "" ]] && ncc config:system:set trusted_domains 11 --value="$pub_ip"
+  pub_ipv4="$(curl -4 -m4 icanhazip.com 2>/dev/null)"
+  pub_ipv6="$(curl -6 -m4 icanhazip.com 2>/dev/null)"
+  [[ "$pub_ipv4"   != "" ]] && ncc config:system:set trusted_domains 11 --value="$pub_ipv4"
+  [[ "$pub_ipv6"   != "" ]] && ncc config:system:set trusted_domains 12 --value="[$pub_ipv6]"
```

Comment:
This syntax is preferrable:
```suggestion
  [[ -z "$pub_ipv6" ]] || ncc config:system:set trusted_domains 12 --value="[$pub_ipv6]"
```